beetsplug/bandcamp/_search.py

```diff
@@ -67,7 +61,13 @@ def get_matches(text: str) -> JSONDict:
     for pat in RELEASE_PATTERNS:
         for m in pat.finditer(text):
             result.update(m.groupdict())
-    result["type"] = result["type"].lower()
+    for url_pat in URL_PATTERNS:
```

Comment:
That was done because at one point if both patterns matched, the later one would override the former, but apparently I fixed that at some point. The tests still work fine when they're in `RELEASE_PATTERNS`, so I'll move them back.